packages/lib-webpack/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  root: true,
+  extends: ['plugin:@monorepo/eslint-plugin-internal/node-typescript-prettier'],
```

Comment:
FYI - we have some issues with our `node-typescript-prettier` preset, which is why the `lint` script in `lib-webpack` is currently disabled.

I'll look into this and fix it via follow-up change.

Reply:
This needs a ticket

Reply:
Ticket: https://issues.redhat.com/browse/HAC-761